nilearn/region.py

```diff
@@ -6,6 +6,7 @@
 """
 # Author: Philippe Gervais
 # License: simplified BSD
+from past.builtins import xrange
```

Comment:
xrange being imported but not used seems to happen quite a few times, can you check all of them?